battleship.py

```diff
@@ -77,7 +77,14 @@ def emptyGrid(rows, cols):
 Returns: 2D list of ints
 '''
 def createShip():
-    return
+    r=random.randint(1,8)
+    c=random.randint(1,8)
+    z=random.randint(0,1)
+    if z==0:

```

Comment:
Can you please check it again. You need to small modification here.

Reply:
changed as per request